### PR TITLE
[SwitchBoard/DIALServer] Configuration Changes.

### PR DESCRIPTION
**Pull request by anjalirajan**
Configuration Changes corresponding to the DIALServer Plug-in and the SwitchBoard Plug-In.

### CODE REVIEW

File: package/netflix/Config.in [hunk 1]
```diff
@@ -1,5 +1,6 @@
 config BR2_PACKAGE_NETFLIX
 	bool "netflix"
+    default n
```
Comment (wouterlucas) [inline package/netflix/Config.in]: I don't think you need to set a default here.

File: package/wpe/wpeframework-dialserver/Config.in [hunk 2]
```diff
@@ -1,5 +1,5 @@
 config BR2_PACKAGE_WPEFRAMEWORK_DIALSERVER
 	bool "DIALServer"
-
+    default n
```
Comment (wouterlucas) [inline package/wpe/wpeframework-dialserver/Config.in]: Same here, no default needed

File: package/wpe/wpeframework-switchboard/Config.in [hunk 3]
```diff
@@ -1,4 +1,5 @@
 config BR2_PACKAGE_WPEFRAMEWORK_SWITCHBOARD
 	bool "SwitchBoard"
+    default n
```
Comment (wouterlucas) [inline package/wpe/wpeframework-switchboard/Config.in]: Same here, no default needed.

**Comment by anjalirajan** [comment]
Removed the default lines from the package configurations.

**Pull request closed without merging**